docs/reference/es_compatible_api.md

```diff
@@ -131,10 +132,47 @@ If a parameter appears both as a query string parameter and in the JSON payload,
 | `from`     | `Integer`   |  The rank of the first hit to return. This is useful for pagination.  |  0  |
 | `query` | `Json object` | Describe the search query. See [Query DSL](#query-dsl) | (Optional) |
 | `size` | `Integer` | Number of hits to return. |  10 |
-| `sort` | `JsonObject[]` | Describes how documents should be ranked. | `[]` |
+| `sort` | `JsonObject[]` | Describes how documents should be ranked. See [Sort order](#sort-order) | `[]` |
 | `aggs` | `Json object` | Aggregation definition. See [Aggregations](aggregation.md). | `{}` | `
 
 
+#### Sort order
+
+You can define up to two criteria on which to apply sort.
+If two criteria are supplied, the lexicographical order is applied.
+In other words, the second criterion will only be used in presence of a tie for the first criterion.
+
+A given criterion can either be
+- the name of a fast field (explicitly defined in the schema or captured by the dynamic mode)
+- `_score` to sort by BM25.
+
+It is also possible to define whether the different component should be taken as descending/ascending using the
```

Comment:
I think I would flip the order here and put the default syntax first. And then I would say something like "The default sort order can be overwritten using the following syntax" and put this example. That will allow us to avoid saying "It is also possible" multiple times and introducing a new term "component" that could be a bit confusing for readers.